hack/import-restrictions.json

```diff
@@ -1,9 +1,228 @@
 [
-   {
-      "baseImportPath" : "pkg/image",
-      "allowedImports" : [
-         "pkg/client",
-         "vendor"
-      ]
-   }
+  {
+    "checkedPackages": [
```

Comment:
This API is a little hard to follow. It's not clear we are implicitly allowing imports of self and of other checked packages